core/src/main/java/com/klaviyo/coresdk/networking/NetworkBatcher.kt

```diff
@@ -46,7 +48,7 @@ object NetworkBatcher {
         override fun run() {
             val emptied = emptyRequestQueue(forceEmpty)
             if (!emptied) {
-                handler?.postDelayed(this, KlaviyoConfig.networkFlushCheckInterval)
+                handler?.postDelayed(this, KlaviyoConfig.networkFlushCheckInterval.toLong())
```

Comment:
This was changed to an `int` above, should we have `toInt()` here? Caveat, I have no idea what I'm talking about since I've never worked with kotlin, or much with java either.

Reply:
`.toInt()` wouldnt be needed since networkFlushCheckInterval is already an `Int` type but this doesnt matter since the method signature for `postDelayed` has this second argument as a `Long` thus the need to cast the `Int` with `.toLong()`